.github/workflows/quic.yml

```diff
@@ -124,8 +124,8 @@ jobs:
       os: ${{ matrix.os }}
       ref: ${{ github.event.client_payload.sha || github.event.client_payload.ref || inputs.ref || 'main' }}
 
-  build-unix:
-    name: Build Unix
+  build-ubuntu20.04:
```

Comment:
Don't do it this way. Just add `ubuntu-22.04` to the `os` part of the matrix below.